protoc-cli/src/main/java/io/ballerina/protoc/protobuf/cmd/GrpcCmd.java

```diff
@@ -125,46 +125,38 @@ private static void exportResource(String resourceName, ClassLoader classLoader)
     }
     
     @Override
-    public void execute() {
-        //Help flag check
-        if (helpFlag) {
-            String commandUsageInfo = BLauncherCmd.getCommandUsageInfo(getName());
-            outStream.println(commandUsageInfo);
-            return;
-        }
+ public void execute() {
+    // Help flag check
+    if (helpFlag) {
+        String commandUsageInfo = BLauncherCmd.getCommandUsageInfo(getName());
+        outStream.println(commandUsageInfo);
+        return;
+    }
 
+    try {
         if (protoPath.endsWith("**.proto") || new File(protoPath).isDirectory()) {
             // Multiple proto files
-            List<String> protoFiles;
-            try {
-                protoFiles = getProtoFiles(protoPath);
-            } catch (IOException e) {
-                String errorMessage = "Failed to find proto files in the directory. " +
-                        "Please input a valid proto files directory.";
-                outStream.println(errorMessage);
-                return;
+            List<String> protoFiles = getProtoFiles(protoPath);
+            if (protoFiles.isEmpty()) {
+                throw new RuntimeException("Input directory does not contain any proto files. Please input a valid proto files directory."); // Improved error message
```

Comment:
```suggestion
                throw new RuntimeException("Input directory does not contain any proto files. Please input a valid proto files directory.");
```

Also if the line length is >120 characters, you may consider breaking it into multiple lines (Applicable for the other places as well).